frontend/src/components/log-in/Login.jsx

```diff
@@ -37,12 +37,6 @@ function Login() {
               >
                 Login
               </Button>
-              <Button
-                className="signin-button button-margin"
-                onClick={handleSignin}
```

Comment:
@athul-rs Can you remove `handleSignin` function as well?

Reply:
@tahierhussain updated